### ISSUE TITLE
I agree to the [terms of service] and will adhere to them unconditionally -- traduction

### ISSUE THREAD

**Issue by dali-rajab** [PrestaShop/PrestaShop#18584]
#### Describe the bug

Hi every one, this may be not a bug, it may be my bad understanding of translations, translations scopes ... but here is my problem
I have the latest fresh installation of prestashop 1.7.6.4, I just added a new commercial theme, then I tryed to translate the sentense that says "I agree to the [terms of service] and will adhere to them unconditionally", so I went to backoffice "International->Traductions", I choosed Theme traductions, then I selected the commercial theme and the desired language, I searched for this string but no results were found !!?? 

Then I went to "International->Traduction", but this time I choosed the classic theme, i found the desired string to translate, i changed it, and guess what ! Boom ! it worked in both themes ???!

#### Expected behavior

For me, designers, developpers and webmasters, the expected translation should be managed in the activated theme, or in both of them, or may be this is some kind of theme heritance, I don't know.

#### Steps to Reproduce

1.  Install any theme in parallel with the classic one, and activate it.
1. Go to "International->Traductions".
2. Choose the new activated theme.
3. search for "I agree to the [terms of service] and will adhere to them unconditionally"
4. no results !!

**Screenshots**
![image](https://user-images.githubusercontent.com/39015823/79015660-874f6300-7b6d-11ea-8b06-1a59ecac94b4.png)

#### Additional information

* PrestaShop version: 1.7.6.4 (fresh installation)

**Comment by alfonsolr09** [comment]
Hello, it also happened to me when trying to translate a word from the cart, I think I remember.
I suppose it will be because if the theme you have does not bring that translation, prestashop will take the translation of the classic theme.

**Comment by khouloudbelguith** [comment]
Hi @dali-rajab,

We are awre of this issue, it is added to our debug roadmap.
Your ticket is a duplicate of #9818
>For a quick fix

You can edit the file:
`\src\PrestaShopBundle\Service\TranslationService.php`
and change here "classic" to "theme_name" (or another theme name) and all default translations and expressions are displayed, but after this custom theme translations in the tab Shop->Theme->Mytheme are not available.

So, I close it.
Thanks!

**Comment by khouloudbelguith** [comment]
Duplicate of #9818
